test/test_no_regression.py

```diff
@@ -0,0 +1,70 @@
+# coding=utf-8
```

Comment:
English (without accentuated character) comment are required. Problably encoding will not be necessary after